correct the location of this file (#1995)
OSC/ondemand: DEVELOPMENT.md

Create and edit files in the host's home directory and to mount in
new configurations.

> Remove `~/.config/ondemand/container/config/ood_portal.yml` to reset
> or edit your container's password.

Author: ```suggestion
Edit or remove `~/.config/ondemand/container/config/ood_portal.yml` to change
you container's password.
```

> Edit or remove `~/.config/ondemand/container/config/ood_portal.yml` to change
> you container's password.

Author: ```suggestion
Edit or remove `~/.config/ondemand/container/config/ood_portal.yml` to change
your container's password.
```
